backend/src/main/java/capstone/backend/mapper/ProductMapper.java

```diff
@@ -0,0 +1,50 @@
+package capstone.backend.mapper;
+
+import capstone.backend.model.db.Product;
+import capstone.backend.model.dto.ProductDTO;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ProductMapper {
+
+    private ProductMapper(){
```

Comment:
is there a reason why the method ist empty?

Reply:
Yes. Since this class has only static methods, there is a private constructor in here to override/hide the default public constructor to prevent initiation of the class